devito/operator/profiling.py

```diff
@@ -35,10 +33,15 @@ class Profiler:
     _default_libs = []
     _ext_calls = []
 
+    _include_dirs = []
+    _lib_dirs = []
```

Comment:
what's the difference between this and the class attribute above `_default_libs` ?